src/server/routes/www.test.js

```diff
@@ -3,12 +3,106 @@ import { useDatabase } from 'server/testUtils'
 import Build from 'server/models/Build'
 import Repository from 'server/models/Repository'
 import Organization from 'server/models/Organization'
+import Screenshot from 'server/models/Screenshot'
 import ScreenshotBucket from 'server/models/ScreenshotBucket'
+import ScreenshotDiff from 'server/models/ScreenshotDiff'
 import www from './www'
 
 describe('GraphQL', () => {
   useDatabase()
 
+  describe('screenshotDiffs', () => {
+    it('should sort the diffs by score', async function () {
+      const organization = await Organization.query().insert({
```

Comment:
We will have to rewrite everything using factories, it's simpler and more concise.